functional-tests/src/test/groovy/listeners/BrowserStackReportingSpec.groovy

```diff
@@ -0,0 +1,29 @@
+package listeners
+
+import geb.spock.GebReportingSpec
+import geb.Browser
+
+class BrowserStackReportingSpec extends GebReportingSpec {
+
+  @Override
+  Browser getBrowser() {
+    Browser _browser = super.getBrowser()
+
+    if (_browser && _browser.driver) {
+      SessionIdHolder.instance.sessionId.set(_browser.driver.getSessionId())
+      _browser.driver.manage().window().maximize()
+    }
+
+
+
+    _browser
+  }
+
+  @Override
+  void resetBrowser() {
+    def driver = browser.driver
```

Comment:
I don't see `browser` defined anywhere here, unless it is in the parent or something?

Reply:
Believe Geb gets the browser implicity, based on configurations in GebConfig